docs/vgclog.md

```diff
@@ -0,0 +1,260 @@
+
+Verbose GC logs, when enabled, begin capturing information as soon as GC is initialized. 
+
+To help you visualize and analyze the GC, you can feed verbose GC log files into various diagnostic tools and interfaces. Examples include tools such as the [Garbage Collection and Memory Visualizer extension for Eclipse](https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-garbage-collection-and-memory-visualizer-gcmv), and online services such as [GCEasy](https://gceasy.io).  
+
+**Note:** You can run one or more GC traces by using the ['-Xtgc' option](xtgc.md) to get more detailed information to help diagnose GC problems or perform finer tuning. 
+
+## How to generate a verbose GC log  
+
+You can enable verbose GC logs by specifying the `-verbose:gc` standard option when you start your application. For more information, see [standard command-line options](cmdline_general.md). 
+
+The output of `-verbose:gc` is printed to STDERR by default. To print the log output to a file, append the [`-Xverbosegclog`](xverbosegclog.md) option. You can also use this option to print to a succession of files, where each file logs a specified number of GC cycles.
+
+## Verbose GC log contents and structure 
+The verbose GC logs are printed in XML format and consist of the following sections:  
+
+- A summary of your GC configuration, which is captured in the `<initialized>` XML element. 
+
+- Information about the GC cycles that ran, including each cycle's GC operations and GC increments.  
+
+For definitions of GC cycles, operations, phases, see [Garbage collection policies](gc.md#garbage-collection-policies). For definitions of GC increments, see [GC increments and interleaving](#gc-increments-and-interleaving).
+
+The logs record when GC cycles and their increments start and end, and list the GC operations that run within these increments to manage or reclaim memory. You can also determine which type of event triggered the cycle or increment, and the amount of memory available to your application before and after processing.  
+
+Not all operations that take place during GC are recorded. For example, the logs record when an increment of a concurrent GC cycle starts and ends, but most concurrent GC operations are not recorded.
+
+### Initialization
+
+The first section of the log records the configuration of the garbage collector, for example:  
+
+```xml
+<initialized id="1" timestamp="2020-10-18T13:27:07.691"> 
+
+    <attribute name="gcPolicy" value="-Xgcpolicy:gencon" /> 
+
+    <attribute name="maxHeapSize" value="0x40000000" /> 
+
+    <attribute name="initialHeapSize" value="0x40000000" /> 
+
+    <attribute name="compressedRefs" value="true" /> 
+
+... 
+ 
+<system> 
+
+    <attribute name="physicalMemory" value="100335456256" /> 
+
+    <attribute name="numCPUs" value="28" /> 
+
+... 
+
+</system> 
+
+<vmargs> 
+
+    <vmarg name="-Xoptionsfile=/java/perffarm/sdks/O11_j9_x64_linux-20201014/sdk/lib/options.default" /> 
+… 
```

Comment:
It's nice to have -Xmx (and even Xms) option here. It's one that's frequently used (a user would specify it via command line options when starting a JVM), and it's good to demonstrate that it will show up in this section.